css/layouts.css

```diff
@@ -0,0 +1,90 @@
+@media all and (min-width: 769px) {
+  body {
+    height: auto;
+    margin: auto;
+    max-width: 960px;
+    min-width: 769px;
+    text-align: center;
+  }
+
+  nav, section {
```

Comment:
by giving your nav and sections all the same border - the border appears thicker where the containers are touching